Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -208,27 +229,27 @@ void OpenALStream::SoundLoop()
   {
     EAXSetBufferMode eaxSetBufferMode;
     eaxSetBufferMode = (EAXSetBufferMode)alGetProcAddress("EAXSetBufferMode");
-    eaxSetBufferMode(numBuffers, uiBuffers, alGetEnumValue("AL_STORAGE_ACCESSIBLE"));
+    eaxSetBufferMode(num_buffers, ui_buffers.data(), alGetEnumValue("AL_STORAGE_ACCESSIBLE"));
     err = CheckALError("setting X-RAM mode");
   }
 #endif
 
   // Generate a Source to playback the Buffers
-  alGenSources(1, &uiSource);
+  alGenSources(1, &ui_source);
   err = CheckALError("generating sources");
 
   // Set the default sound volume as saved in the config file.
-  alSourcef(uiSource, AL_GAIN, fVolume);
+  alSourcef(ui_source, AL_GAIN, fVolume);
 
   // TODO: Error handling
   // ALenum err = alGetError();
 
-  unsigned int nextBuffer = 0;
-  unsigned int numBuffersQueued = 0;
-  ALint iState = 0;
+  unsigned int next_buffer = 0;
+  unsigned int num_buffers_queued = 0;
+  ALint i_state = 0;
```

Comment:
No need to stick to Hungarian notation, especially since it looks weird in underscore_case 😆